transport-parent/transport-netty/pom.xml

```diff
@@ -21,6 +21,7 @@
     <dependency>
       <groupId>io.projectreactor.netty</groupId>
       <artifactId>reactor-netty</artifactId>
+      <version>1.0.0</version>
```

Comment:
Please revert it. Change the reactor-bom version in the main pom.xml:42 instead.

```
<reactor.version>2020.0.4</reactor.version>
```

Reply:
Ahh, I was looking for what defined that version but couldn't find it. I'll update that, thanks.